sdc/tests/test_series.py

```diff
@@ -3498,8 +3641,8 @@ def test_impl(S):
         for n in range(1, 5):
             S = pd.Series(['a', 'ab', 'abc', 'c', 'f', 'hh', ''] * n)
             start, end = get_start_end(len(S))
-            pd.testing.assert_series_equal(hpat_func(S[start:end]), test_impl(S))
-            self.assertTrue(count_array_OneDs() > 0)
+            pd.testing.assert_series_equal(hpat_func(S[start:end]), test_impl(S[start:end]))
+            # self.assertTrue(count_array_OneDs() > 0)
```

Comment:
I don't think it's a good idea to remove the check that array was distributed. This test when run in the old pipeline should not be affected by adding impl via new one.